WEEKLY_UPDATES.md

```diff
@@ -1,5 +1,27 @@
 # Weekly Updates
 
+## 2023-02-07
+
+The last two weeks we've been focusing on working towards the 1.7.0 release, and on going through our issue backlog.
+
+This weekend some of us have been to FOSDEM and had a stand there - it was amazing to hear in person how many people have already migrated to OpenTofu! We've also distributed
```

Comment:
"We've also distributed" what?

Reply:
Looks like a copilot completion I thought I've applied.